<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color:rgb(153, 193, 241);
  border: 1px solid gray;
}</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>13</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 13pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Function =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 13pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>X min =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 13pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>X max  =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 13pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Y min =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 13pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Y max =</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_x_min">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
	border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_y_min">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
	border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_x_max">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
	border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_y_max">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
	border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fun">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>13</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 13pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
   border: 1px solid gray; 
   font: 25 11pt &quot;Umpush&quot;;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>PLOT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_msg">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>13</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 13pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
